.github/workflows/label-community-pr.yml

```
@@ -0,0 +1,28 @@
+name: Label community PRs
+
+on:
+  schedule:
+    - cron: '*/30 * * * *'
+
+permissions:
+  pull-requests: write
+
+jobs:
+  check_author:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check PR Author Membership
```

We'd better name it to reflect our purpose, rather than our means.
```suggestion
      - name: Label Community PR
```

---

Agreed. I reuse the pr that Hans closed and forget to rename it. Fixed.